lib/virt_autotest/virtual_network_utils.pm

```diff
@@ -100,6 +100,11 @@ sub test_network_interface {
 
     save_guest_ip("$guest", name => $net);
 
+    # ENHANCEMENT:Wait until 15SP5 PV guest's new network interface is workable
+    # Please refer to poo#121351 and poo#121348 for more details
+    script_retry("ping -c10 $guest", delay => 6, retry => 30) if ($guest =~ m/sles-15-sp5-64-pv/i);
```

Comment:
why not increase the retry of line 112 followed directly? that line will wait 9 minutes to access guest over ssh. how long does it takes for the sles-15-sp5-64-pv guest?